script/Ciel/Day 13/13_01_01_CIEL13_5.txt

```diff
@@ -21,12 +23,14 @@ A vampire.
 [sha:ff2f863803e9127018c5e8c26b61e78af09d6223]{
 -- Page 513, Offset 34759.
 -- ……そう、アレは吸血鬼だ。
-...That's right. That thing is a vampire.
+Yeah... that thing is a vampire alright.
 }
 [sha:ff19e2ad42f7d76338e77f586cacfbdbf340f709]{
 -- Page 513, Offset 34760.
 -- <判|わか>りたくもないが、今の俺は同属として、アレがどれほどの吸血鬼なのか感じ取れる。
-I'd really prefer not to know, but now that I'm also a vampire, I'm capable of discerning her Rank.
+I'd have preferred blissful ignorance,
+ but now that I'm a vampire myself,
+ I'm able to tell at a glance what Rank she is.
```

Comment:
I mean I agree with the interpretation, but that feels like an unnatural way of phrasing it. If you don't like the ignorance part of the 'blissful ignorance' set phrase, how about 
> Now that I'm a vampire myself, I can tell at a glance what Rank she is――even though I wish I couldn't

or something like that

Reply:
I just think the implication that he's able to tell someone's rank because he's a vampire is interesting, and I want that to come through quite explicitly.